src/Infrastructure/Entity/WorkflowTransition.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Entity;
+
+use DateTimeInterface;
+use SprykerSdk\Sdk\Core\Domain\Entity\WorkflowTransition as EntityWorkflowTransition;
+use SprykerSdk\SdkContracts\Entity\WorkflowInterface;
+
+class WorkflowTransition extends EntityWorkflowTransition
+{
+    /**
+     * @var int
+     */
+    protected int $id;
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param \SprykerSdk\SdkContracts\Entity\WorkflowInterface $workflow
+     *
+     * @return $this
+     */
+    public function setWorkflow(WorkflowInterface $workflow)
+    {
+        $this->workflow = $workflow;
+
+        return $this;
+    }
+
+    /**
+     * @param array $status
+     *
+     * @return $this
+     */
+    public function setStatus(array $status)
+    {
+        $this->status = $status;
+
+        return $this;
+    }
+
+    /**
+     * @param string|null $transition
+     *
+     * @return $this
+     */
+    public function setTransition(?string $transition)
```

Comment:
can it be null?

Reply:
It could be null in previous implementation, I forgot to change

Reply:
![image](https://user-images.githubusercontent.com/5063777/174271424-92161826-e91b-443d-98ea-e353773ab5fe.png)